### PR TITLE
Array binary search

### PR DESCRIPTION
**Pull request by littl3birdy**

### CODE REVIEW

**Comment by littl3birdy** [comment]
- [x ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ x] README for this challenge is complete
     - [x ] Summary, Description, Approach & Efficiency, Solution
     - [ x] Link to code
     - [ x] Picture of whiteboard

File: code401challenges/src/main/java/code401challenges/BinarySearch.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            }if (valueToMatch > sortedArr[start]) {
+                mid = start + ((end-start)/2);
+                start = mid;
+            }else {
+                mid = start - ((start-end)/2);
+                start = mid;
+            }
+
+        }
+    }
```
Comment (SharinaS) [inline code401challenges/src/main/java/code401challenges/BinarySearch.java]: You'll need to remove one of these curly brackets (you've got too many), and add a return statement that says something like `return -1`.
